.eslintrc.js

```diff
@@ -70,6 +70,18 @@ module.exports = {
     'no-template-curly-in-string': 'error',
     'no-useless-catch': 'error',
     'no-useless-concat': 'error',
+    'prefer-destructuring': ['error', {
+      'VariableDeclarator': {
+        'array': false,
+        'object': true,
+      },
+      'AssignmentExpression': {
+        'array': false,
+        'object': false,
+      },
+    }, {
+      'enforceForRenamedProperties': false,
```

Comment:
Why not `true`? 🤔

Reply:
Enabling it seems to conflict with turning off `VariableDeclarator.array` and `AssignmentExpression.array` which in turn raises annoying issues, e.g.:

```js
const contract = contractMap[address]
```

Reply:
Huh, writing that out makes me realize that that isn't a conflict, that isn't array access. Fair enough.

Enabling it raises 1k+ issues that I'm too lazy to fix by hand.

Reply:
It is more verbose, though:

```diff
-const contract = contractMap[address]
+const { [address]: contract } = contractMap
```